yarn.lock

```diff
@@ -3510,6 +3570,11 @@ string_decoder@~1.0.3:
   dependencies:
     safe-buffer "~5.1.0"
 
+stringstream@^0.0.6:
+  version "0.0.6"
+  resolved "https://registry.yarnpkg.com/stringstream/-/stringstream-0.0.6.tgz#7880225b0d4ad10e30927d167a1d6f2fd3b33a72"
+  integrity sha512-87GEBAkegbBcweToUrdzf3eLhWNg06FJTebl4BVJz/JgWy8CvEr9dRtX5qWphiynMSQlxxi+QqN0z5T32SLlhA==
+
 stringstream@~0.0.4:
   version "0.0.5"
```

Comment:
The version here is still `0.0.5`, so I suspect that specifying `stringstream` as a package in `package.json` won't end up resolving the security alert. The security alert's [automated fixer's PR](https://github.com/execonline-inc/vimeo-upload-lambda/pull/7/files) simply modified the `yarn.lock` directly, so that may be all that needs to be done.